tracer/src/Datadog.Trace.SourceGenerators/Directory.Build.props

```diff
@@ -0,0 +1,19 @@
+<Project>
+  <!-- Deliberately not importing parent directory props-->
+
+  <PropertyGroup>
+    <!-- Strong name signature -->
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../../../Datadog.Trace.snk</AssemblyOriginatorKeyFile>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- StyleCop -->
+    <Compile Include="../GlobalSuppressions.cs" Link="GlobalSuppressions.src.cs" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.*" PrivateAssets="all" />
+    <AdditionalFiles Include="../../stylecop.json" Link="stylecop.json" />
+    <Compile Include="../../GlobalSuppressions.cs" Link="GlobalSuppressions.solution.cs" />
+  </ItemGroup>
```

Comment:
Given we need these even when we don't want to inherit directory.props in general, I wonder if it's worth moving them to a separate props file we can explicitly import instead of copy-pasting? 🤔

Reply:
Good idea. I noticed the props file in `/tracer` is good for this, so for these two projects, I skipped importing `/tracer/src` and went for the parent's parent instead.